Gems/Atom/RPI/Code/Source/RPI.Builders/Model/MaterialAssetBuilderComponent.cpp

```diff
@@ -219,6 +225,8 @@ namespace AZ
                 }
             }
 
+            bool includeMaterialPropertyNames = true;
```

Comment:
This isn't used, it needs to be passed into CreateMaterialAsset.
(Make sure you test your changes with the flag in MaterialComponentSettings both on and off).

Reply:
Fixed! Did more testing to make sure everything works.